### PR TITLE
Fix setting array null values to None. Add tests to test tables. Support NULLs in table_load_columnar.

### LINKED ISSUE #254: OmniSciDB uses scalar null values to set array null values
Related to #245 and #240.

```sql
omnisql> create table test (i2 SMALLINT[]);
omnisql> insert into test values ({});
omnisql> insert into test values ({NULL});
omnisql> insert into test values ({2});
omnisql> insert into test values ({1, NULL, 3});
omnisql> insert into test values (NULL);
omnisql> select * from test;
i2
{}
{NULL}
{2}
{1, NULL, 3}
NULL
```

from RBC:
```python
>>> _, result = omnisci.sql_execute(f'select i2 from test;')
>>> print(list(result))
[([],), ([-32768],), ([2],), ([1, -32768, 3],), (None,)]
```

`-32768` is the null value for `int16` and not `Array<int16>`. The same happens for every int/float type.

### PR DESCRIPTION
**Pull request by pearu**
Fixes #254, #261

### CODE REVIEW

**Review by guilhermeleobas — APPROVED** [review]
LGTM. I just wonder if changing `_make_row_results_set` wouldn't affect older RBC connected to older versions of omniscidb.

File: rbc/omniscidb.py [hunk 1]
```diff
@@ -405,32 +424,43 @@ def load_table_columnar(self, table_name, **columnar_data):
...
-                for row in column_data:
+                arrays = []
+                for arr in column_data:
+                    if arr is None:
+                        arr_nulls = None
+                    else:
+                        arr_nulls = [v is None for v in arr]
+                        if True in arr_nulls:
+                            # TODO: null support for text
+                            null_value = self._null_values[datumtype_map[datumtype]]
```
[the remark this answers is not in the document]
Comment (pearu) [reply rbc/omniscidb.py:460]: Good point. Fixed by backporting load_table_columnar NULL support. Tested with cpu-only omniscidb 5.4.1 and 5.2.2.

**Comment by pearu** [comment]
> LGTM. I just wonder if changing `_make_row_results_set` wouldn't affect older RBC connected to older versions of omniscidb.

Thanks for the review. I have tested this PR with omniscdb 5.5.0, 5.4.1, and 5.2.2, so the changes are ok.

**Comment by guilhermeleobas** [comment]
LGTM! Do you know what server versions are available on conda-forge? I can send a PR adding them to Travis CI.

**Pull request merged**